src/specsynthase/specbuilder.py

```diff
@@ -5,10 +5,7 @@
 class SpecBuilder(OrderedDict):
     def __init__(self, *args, **kwargs):
         OrderedDict.__init__(self, *args, **kwargs)
-        self._paths_key = "paths"
-        self._definitions_key = "definitions"
-        self._parameters_key = "parameters"
-        self._response_key = "responses"
+        self._mergeable_keys = ["definitions", "parameters", "paths", "responses", "securityDefinitions"]
```

Comment:
I wonder if we can get rid of this altogether once #3 is fixed (and instead, simply merge all keys that are present). That would future proof the code better. We'd need to figure out what to do about keys that present in multiple files (do we throw an error, or overwrite) if we did that.